tests/test_validate_path.py

```diff
@@ -0,0 +1,189 @@
+
+result = requests.get('http://localhost:5000/01001')
+assert(result.json() == {'extenso': 'mil e um'})
+
+result = requests.get('http://localhost:5000/01010')
+assert(result.json() == {'extenso': 'mil e dez'})
+
+result = requests.get('http://localhost:5000/01011')
+assert(result.json() == {'extenso': 'mil e onze'})
+
+result = requests.get('http://localhost:5000/01020')
+assert(result.json() == {'extenso': 'mil e vinte'})
+
+result = requests.get('http://localhost:5000/01025')
+assert(result.json() == {'extenso': 'mil e vinte e cinco'})
+
+result = requests.get('http://localhost:5000/01100')
+assert(result.json() == {'extenso': 'mil e cem'})
+
+result = requests.get('http://localhost:5000/01101')
+assert(result.json() == {'extenso': 'mil cento e um'})
+
+result = requests.get('http://localhost:5000/01110')
+assert(result.json() == {'extenso': 'mil cento e dez'})
+
+result = requests.get('http://localhost:5000/01115')
+assert(result.json() == {'extenso': 'mil cento e quinze'})
+
+result = requests.get('http://localhost:5000/01120')
+assert(result.json() == {'extenso': 'mil cento e vinte'})
+
+result = requests.get('http://localhost:5000/01125')
+assert(result.json() == {'extenso': 'mil cento e vinte e cinco'})
+
+result = requests.get('http://localhost:5000/01200')
+assert(result.json() == {'extenso': 'mil e duzentos'})
+
+result = requests.get('http://localhost:5000/01201')
+assert(result.json() == {'extenso': 'mil duzentos e um'})
+
+result = requests.get('http://localhost:5000/02000')
+assert(result.json() == {'extenso': 'dois mil'})
+
+result = requests.get('http://localhost:5000/02001')
+assert(result.json() == {'extenso': 'dois mil e um'})
+
+result = requests.get('http://localhost:5000/02010')
+assert(result.json() == {'extenso': 'dois mil e dez'})
+
+result = requests.get('http://localhost:5000/02100')
+assert(result.json() == {'extenso': 'dois mil e cem'})
+
+result = requests.get('http://localhost:5000/02101')
+assert(result.json() == {'extenso': 'dois mil cento e um'})
+
+result = requests.get('http://localhost:5000/02101')
+assert(result.json() == {'extenso': 'dois mil cento e um'})
+
+result = requests.get('http://localhost:5000/02201')
+assert(result.json() == {'extenso': 'dois mil duzentos e um'})
+
+result = requests.get('http://localhost:5000/11201')
+assert(result.json() == {'extenso': 'onze mil duzentos e um'})
+
+
+result = requests.get('http://localhost:5000/20201')
+assert(result.json() == {'extenso': 'vinte mil duzentos e um'})
+
+result = requests.get('http://localhost:5000/25201')
+assert(result.json() == {'extenso': 'vinte e cinco mil duzentos e um'})
+
+result = requests.get('http://localhost:5000/10000')
+assert(result.json() == {'extenso': 'dez mil'})
+
+result = requests.get('http://localhost:5000/-00000')
+assert(result.json() == {'extenso': 'zero'})
+
+result = requests.get('http://localhost:5000/-99999')
+assert(result.json() == {'extenso': 'menos noventa e nove mil novecentos e noventa e nove'})
```

Comment:
esperado: menos noventa e nove mil _**e**_ novecentos e noventa e nove
(tem vários acima tbm que nem comentei)

Reply:
Sendo sincero, eu fui pela tradução que o google me dava ( https://www.google.com/search?q=99999+por+extenso ). Vi alguns lugares usando o "e" e outros não, mas vou confimar o correto em algum lugar confiável e arruma o código.

Reply:
O correto é o que tá nos requisitos (;

Reply:
you've got a point =)